plugins/catalog-backend-module-github/src/lib/org.ts

```diff
@@ -52,8 +52,17 @@ export function buildOrgHierarchy(groups: GroupEntity[]) {
 // Ensure that users have their direct group memberships.
 export function assignGroupsToUsers(
   users: UserEntity[],
-  groupMemberUsers: Map<string, string[]>,
+  groups: GroupEntity[],
 ) {
+  const groupMemberUsers = new Map(
+    groups.map(group => {
+      const groupKey = group.metadata.namespace
```

Comment:
could do `group.metadata.namespace && group.metadata.namespace !== DEFAULT_NAMESPACE` here if you want to be thorough

Reply:
Included